src/main/java/org/broadinstitute/consent/http/service/DaaService.java

```diff
@@ -101,11 +107,38 @@ public void removeDacFromDaa(Integer dacId, Integer daaId) {
     daaDAO.deleteDacDaaRelation(dacId, daaId);
   }
 
-  public List<DataAccessAgreement> findAll() {
-    List<DataAccessAgreement> daas = daaDAO.findAll();
-    if (daas != null) {
-      return daas;
+  public boolean isBroadDAA(int daaId, List<DataAccessAgreement> allDaas, List<Dac> allDacs) {
```

Comment:
I think we could use some additional documentation/TODO/ticket reference here to indicate that this is not the permanent solution to this problem.

Reply:
Is it not the permanent solution to the problem? I know this isn't an ideal solution, but I don't think we had an idea for a more "permanent" solution if I'm not mistaken.

Reply:
Sorry I missed this response. A more permanent solution would be to add a database column to the DAC table that defines this and allow for admins to set this in the UI or by API. There are other potential implementations, but we shouldn't be relying on names/creation dates to infer this value.

Reply:
I agree with @rushtong . Instead of a boolean I would recommend an `enum` column that can be set to Broad, and the code should check that instead.

Reply:
@rushtong @fboulnois Should I implement those changes in this PR itself or create a new ticket to include this additional scoping? I'm not sure what we want to do in alignment with our "v1" of this work.

Reply:
My suggestion is just to document the better approach. This approach is OK for now. The ideal solution requires more time and effort that we have available right now.

Reply:
That seems reasonable. I will unblock approval to be able to move this PR forward.

Reply:
Ticket: https://broadworkbench.atlassian.net/browse/DCJ-498